index.js

```diff
@@ -26,11 +24,17 @@ function parseDatesFromArgv() {
     const startDate = process.argv[2];
     const endDate = process.argv[3];
 
-    const startMoment = moment.tz(startDate, timeZone).startOf('day');
+    const startMoment = DateTime.fromISO(startDate, {
+        zone: timeZone || 'utc',
+        setZone: true,
+    });
 
-    const endMoment = moment.tz(endDate, timeZone).endOf('day');
+    const endMoment = DateTime.fromISO(endDate, {
+        zone: timeZone || 'utc',
+        setZone: true,
+    });
 
-    return [startMoment, endMoment];
+    return [startMoment.toISO(), endMoment.toISO()];
```

Comment:
If we avoid converting toISO here, we could avoid some fromISO calls later. Is there a compelling reason to store these values as strings rather than luxon values that correspond to what used to be moment objects?

Reply:
Done